### PR TITLE
feat(sdk): add pages to delete service account and token

### PR DESCRIPTION
**Pull request by rsbh**

### CODE REVIEW

File: sdks/js/packages/core/react/components/organization/api-keys/delete.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          refetch: true
+        }
+      });
+      toast.success('Service account deleted');
+    } catch (err: any) {
+      toast.error('Unable to delete service account', {
+        description: err?.message
+      });
+    } finally {
+      setIsLoading(true);
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/delete.tsx:33]: Shouldn't setLoading be set to false?

File: sdks/js/packages/core/react/components/organization/api-keys/service-user/delete.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          refetch: true
+        }
+      });
+      toast.success('Service account key revoked');
+    } catch (err: any) {
+      toast.error('Unable to revoke service account key', {
+        description: err?.message
+      });
+    } finally {
+      setIsLoading(true);
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/service-user/delete.tsx:37]: This as well?

File: sdks/js/packages/core/react/components/organization/api-keys/service-user/styles.module.css [hunk 3]
```diff
@@ -60,3 +60,27 @@
...
+  z-index: 55;
+  background-color: rgba(104, 112, 118, 0.5);
+}
+
+.addDialogForm {
+  padding: var(--rs-space-5) var(--rs-space-7);
+}
+
+.addDialogFormContent {
+  padding: var(--space-9) var(--space-7);
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/service-user/styles.module.css:81]: Please use new vars here as well.

File: sdks/js/packages/core/react/components/organization/api-keys/delete.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+          <Button
+            variant="danger"
+            size="normal"
+            data-test-id="frontier-sdk-delete-service-account-confirm-btn"
+            loading={isLoading}
+            disabled={isLoading}
+            onClick={onDeleteClick}
+            loaderText="Deleting..."
+          >
+            I Understand and Delete
```
Comment (paanSinghCoder) [inline sdks/js/packages/core/react/components/organization/api-keys/delete.tsx:98]: I think we can reframe this sentence better?

Comment (rsbh) [reply sdks/js/packages/core/react/components/organization/api-keys/delete.tsx:98]: This is as per design
Screenshot 2024-11-29 at 12 07 23 PM

**Review by paanSinghCoder — APPROVED** [review]

**Pull request merged**